client/app/dedicated/server/interfaces/ola-configuration/ola-configuration.html

```diff
@@ -0,0 +1,188 @@
+                </div>
+            </div>
+        </oui-field>
+        <oui-field data-label="{{:: 'dedicated_server_interfaces_ola_field_name_label' | translate}}"
+            data-size="l">
+            <input class="oui-input"
+                name="name"
+                type="text"
+                autocomplete="off"
+                placeholder="{{:: 'dedicated_server_interfaces_ola_field_name_placeholder' | translate }}"
+                data-ng-model="$ctrl.configuration.name"
+                data-required>
+        </oui-field>
+    </oui-step-form>
+    <oui-step-form
+        data-header="{{:: 'dedicated_server_interfaces_ola_step2_title' | translate }}"
+        data-description="{{:: 'dedicated_server_interfaces_ola_step2_description' | translate }}"
+        data-valid="!!$ctrl.selectedInterfaces.length && $ctrl.selectedInterfaces.length > 1"
+        data-prevent-next>
+
+        <!-- Obsolete bandwidth option -->
+        <oui-message class="mb-3"
+            data-ng-if="$ctrl.hasObsoleteBandwithOption()"
+            data-type="warning">
+            <p class="mb-0" data-translate="dedicated_server_interfaces_ola_alert_obsolete_option"></p>
+        </oui-message>
+
+        <!-- Forbidden interface -->
+        <oui-message class="mb-3"
+            data-ng-if="$ctrl.notAllowedInterfaces.length"
+            data-type="warning">
+            <p data-translate="dedicated_server_interfaces_ola_alert_not_allowed"></p>
+            <ul class="mb-0">
+                <li
+                    data-ng-repeat="interface in $ctrl.notAllowedInterfaces"
+                    data-ng-bind="interface.name">
+                </li>
+            </ul>
+        </oui-message>
+
+        <oui-datagrid
+            data-rows="$ctrl.interfaces"
+            data-on-row-select="$ctrl.onRowSelect($rows)"
+            data-selectable-rows>
+            <oui-column data-title=":: 'dedicated_server_interfaces_ola_column_name' | translate"
+                data-property="name">
+            </oui-column>
+            <oui-column data-title=":: 'dedicated_server_interfaces_ola_column_current_type' | translate"
+                data-property="type">
+                <span data-translate="{{ 'dedicated_server_interfaces_type_' + $value + '_label' }}"></span>
+            </oui-column>
+            <oui-column data-title=":: 'dedicated_server_interfaces_ola_column_network_offer' | translate">
+                <span class="oui-status oui-status_warning"
+                    data-ng-if="$row.isPublic() && $ctrl.hasObsoleteBandwithOption()">
+                    <span class="oui-icon oui-icon-warning_circle oui-color-warning-medium font-inherit"></span>
+                    <span data-ng-bind="'server_bandwidth_type_' + $ctrl.specifications.bandwidth.type | translate"></span>
+                </span>
+                <span
+                    data-ng-if="$row.isPublic() && $ctrl.specifications.bandwidth.type === 'included'"
```

Comment:
Good use of model class, should avoid test in html.

```suggestion
                    data-ng-if="$row.isPublic() && $ctrl.specifications.isBandwidthIncluded()"
```